docs/installation.rst

```diff
@@ -72,18 +75,26 @@ local `wheel cache`_.
 Building cryptography on Linux
 ------------------------------
 
+.. note::
+
+    If you are on RHEL/CentOS/Fedora/Debian/Ubuntu or another distribution
+    derived from the preceding list, then you should **upgrade pip** and
```

Comment:
Could you please mention that users should use a virtual env and update it? They can easily break their entire OS by updating setuptools and pip as root.

Debian/Ubuntu users have to install python3-venv first.